libhb/muxavformat.c

```diff
@@ -438,7 +438,24 @@ static int avformatInit( hb_mux_object_t * m )
                 }
             }
         } break;
+        case HB_VCODEC_FFMPEG_VCE_AV1:
```

Comment:
This code looks like a duplicate of the existing one for HB_VCODEC_SVT_AV1 and HB_VCODEC_FFMPEG_NVENC_AV1 a few line above.